Misc/NEWS

```diff
@@ -806,6 +806,8 @@ Documentation
 Build
 -----
 
+- Issue #29572: Update Windows build and OS X installers to use OpenSSL 1.0.2k.
```

Comment:
That's fine.  The additional PR is coming. Although aren't we using bpo- stylings now in Misc/NEWS?